public/locales/pt/common.json

```diff
@@ -1,64 +1,64 @@
 {
-  "scrobble": "Scrobble",
-  "getScrobbling": "Iniciar o scrobbling",
-  "loading": "Loading...",
+  "scrobble": "Fazer scrobble",
+  "getScrobbling": "Comece a fazer scrobble",
+  "loading": "Carregando...",
   "artist": "Artista",
-  "song": "Som",
+  "song": "Música",
   "title": "Título",
   "album": "Álbum",
-  "scrobbleFromAlbum": "Scrobble albums",
-  "scrobbleAlbum": "Scrobble album",
-  "album_plural": "Albums",
-  "goBack": "Go back",
-  "topAlbumsBy": "Albums by {{nameOfArtist}}",
-  "noAlbumsFound": "No albums found for <1>{{albumOrArtist}}</1>",
-  "noArtistsFound": "No artists found for <1>{{albumOrArtist}}</1>",
-  "emptyAlbum": "This album appears to be empty.",
-  "scrobbleSongs": "Scrobble songs",
-  "scrobbleSelected": "Scrobble selected",
-  "findAlbumCopy": "Enter an album or artist name",
-  "customTimestamp": "Custom timestamp",
-  "albumTimestampLogicDescription": "Scrobbling tracks \"now\" will scrobble the last track at the current time, backdating the previous ones accordingly (as if you had just finished listening). The \"custom timestamp\" will define the timestamp of the first track and add time to the following tracks from there (i.e. you specify the time you started listening to this album)",
+  "scrobbleFromAlbum": "Fazer scrobble de álbuns",
+  "scrobbleAlbum": "Fazer scrobble do álbum",
```

Comment:
Can we make these options shorter? These texts are used in the menu and it will probably break in small displays.

Reply:
I see. `Scrobble álbuns` would work... people would understand, though the phrase wouldn't make much sense, because we don't read `scrobble` as a verb.

Reply:
ok, we can keep them as they are! I was checking the code and this is used for the home buttons, so they will show correctly anyway :+1: